syft_tensorflow/hook/hook.py

```diff
@@ -98,10 +104,94 @@ def _hook_native_tensor(self, tensor_type: type, syft_type: type):
         self._hook_properties(tensor_type)
 
         # Overload auto overloaded with Torch methods
-        self._add_methods_from_native_tensor(tensor_type, syft_type)
+        exclude = [  # Overload auto overloaded with Torch methods
```

Comment:
can we clean these comments up please 🙏

Reply:
This should be cleaned up by #16 if not, I will do it.